### ISSUE TITLE
Can we map LA's "Food Deserts"?

### ISSUE THREAD

**Issue by lafoodbank** [maptimeLA/food_bank_data_bank#48]
The USDA defines Food Deserts as:
1. They qualify as "low-income communities", based on having: a) a poverty rate of 20 percent or greater, OR b) a median family income at or below 80 percent of the area median family income; AND
2. They qualify as "low-access communities", based on the determination that at least 500 persons and/or at least 33% of the census tract's population live more than one mile from a supermarket or large grocery store (10 miles, in the case of non-metropolitan census tracts).

**Comment by WalkieTalkieLA** [comment]
Yes, it looks like the USDA provides data on "Food Deserts" through their Food Access Research Atlas Data file, and archived Food Desert locator available here: http://www.ers.usda.gov/data-products/food-access-research-atlas/download-the-data.aspx

Another datasource that might also be helpful for comparison is the Centers for Disease Control and Prevention's data on the modified Retail Food Environment Index (mRFEI), which is basically a measurement showing the percentage of food retailers that are considered "healthy". The data are available here: http://www.cdc.gov/obesity/resources/reports.html (from the link scroll down or search for mRFEI). 

You can see a map of the USDA's food deserts here: http://bit.ly/1VwXi0C on HealthyCity.org, and a map of the CDC's mRFEI here: http://bit.ly/1VwWWqN.

Also, many are now looking at "food swamps" instead of "food deserts." Here is an article talking about the difference: http://www.nytimes.com/2012/04/18/health/research/pairing-of-food-deserts-and-obesity-challenged-in-studies.html?_r=0

**Comment by JRHutson** [comment]
@WalkieTalkieLA Thanks for the info. The archived data file you mentioned from USDA has been added to the repo. The CDC data looks awesome. Have to take a look at that as well.

**Comment by JRHutson** [comment]
Uploaded the table of mRFEI scores as well.

**Comment by JRHutson** [comment]
Combine effort with Issue #49

**Comment by kmw** [comment]
enteractive map
http://www.ers.usda.gov/data-products/food-access-research-atlas/go-to-the-atlas.aspx
enter los angeles in the search field (top left corner)
see 2 screen captures (below)
![foodaccessresearchatlaslacitymicro](https://cloud.githubusercontent.com/assets/702795/9676296/30ed994a-527e-11e5-9156-a225aab8fb59.png)
![foodaccessresearchatlaslacitymacro](https://cloud.githubusercontent.com/assets/702795/9676297/30ef3a5c-527e-11e5-97ab-df3e151535ba.png)
